src/ort_genai.h

```diff
@@ -84,12 +91,21 @@ struct OgaSequences : OgaAbstract {
     return OgaSequencesCount(this);
   }
 
+#if __cplusplus >= 202002L
   std::span<const int32_t> Get(size_t index) const {
```

Comment:
Hmm, maybe just have separate 'GetData' and 'GetCount' functions that always exist, and just make the span one not exist with less than C++20?